bdtopo/src/main/groovy/org/orbisgis/geoclimate/bdtopo/InputDataFormatting.groovy

```diff
@@ -97,48 +97,152 @@ String formatBuildingLayer(JdbcDataSource datasource, String building, String zo
                      "Annexe"                            : ["annex": "building"],
                      "Industriel, agricole ou commercial": ["commercial": "commercial"],
                      "Bâtiment"                          : ["building": "building"],
-                     "Industrie lourde"                  : ["heavy_industry": "industrial"]
+                     "Industrie lourde"                  : ["industrial": "industrial"]
```

Comment:
Why not keep heavy_industry type ?

Reply:
If we want to follow the OSM types heavy industry is not defined, it has to be defined later on (as it is the case for the LCZ case)